bundles/org.openhab.binding.flume/src/main/java/org/openhab/binding/flume/internal/actions/FlumeDeviceActions.java

```diff
@@ -90,31 +88,26 @@ public void setThingHandler(@Nullable ThingHandler handler) {
             logger.warn("queryWaterUsage called with null inputs");
             return null;
         }
-
+        if (untilDateTime.isBefore(sinceDateTime)) {
```

Comment:
What if they are equal?

Reply:
I'm okay to require them to be different as a query for water usage with a delta time of 0 doesn't make sense.

Reply:
Yes, that is my point. If they are equal, this block is not executed and the method is not returned early. Instead the query continues.

Reply:
make sense - i updated.